app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,16 @@
+class GuestsController < ApplicationController
+
+  def show
+    @guest = Guest.find(params[:id])
+    @rooms_stayed_in = @guest.rooms
+  end
```

Comment:
You can eliminate line 5 since you can get rooms off of the guest object that you're already passing to the view. We want to send as few objects to the view as possible.